### ISSUE TITLE
Disable plugin if solr isnt part of the project being tested

### ISSUE THREAD

**Issue by jayvdb** [Princeton-CDH/parasolr#41]
On a project unrelated to solr I am getting an error because the plugin doesnt deactive if it doesnt detect solr is configured.  It should at least check INSTALLED_APPS, and at worst give a warning unless the plugin was explicitly enabled.

```py
/usr/lib/python3.8/site-packages/parasolr/pytest_plugin.py:29: in get_test_solr_config
    test_config = settings.SOLR_CONNECTIONS['default'].copy()
/usr/lib/python3.8/site-packages/django/conf/__init__.py:77: in __getattr__
    val = getattr(self._wrapped, name)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <UserSettingsHolder>, name = 'SOLR_CONNECTIONS'

    def __getattr__(self, name):
        if not name.isupper() or name in self._deleted:
            raise AttributeError
>       return getattr(self.default_settings, name)
E       AttributeError: module 'django.conf.global_settings' has no attribute 'SOLR_CONNECTIONS'

/usr/lib/python3.8/site-packages/django/conf/__init__.py:205: AttributeError
```

**Comment by rlskoeser** [comment]
Thanks for documenting; this seems like an easy enough fix. I'm surprised it gets loaded when it isn't even in INSTALLED_APPS, but maybe pytest loads things differently than django does.

**Comment by jayvdb** [comment]
Ya, pytest entry points are all loaded at the beginning, and need to delay any serious work until a pytest hook has called the plugin to configure itself, and then it needs to be sure the plugin is wanted.

**Comment by rlskoeser** [comment]
Part of this may be because I have the plugin defined with `autouse=True` — but that still seems reasonable to me, once I add a check that it's in INSTALLED_APPs as you suggest.

I just pushed a hotfix branch `hotfix/0.5.4`   — do you want to see if it solves your problem?

**Comment by rlskoeser** [comment]
Fixed in version 0.5.4.
